Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.10-slim
+
+WORKDIR /app
+
+RUN python3 -m pip install --upgrade pip
+
+RUN apt-get update && \
+    apt-get install -y git && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN git clone https://github.com/microsoft/TaskWeaver.git
+RUN cd TaskWeaver && pip install -r requirements.txt
```

Comment:
Should we use the file in current folder as context and copying them into docker container when building image? Otherwise how we could load the changes in local if there is any changes made.